openforcefield/tests/test_forcefield.py

```diff
@@ -256,27 +255,30 @@ def test_parameterize_ethanol_different_reference_ordering_openeye(self):
         assert serialized_1 == serialized_2
 
 
-    @pytest.mark.skipif( not(OpenEyeToolkitWrapper.toolkit_is_available()), reason='Test requires OE toolkit')
+    @pytest.mark.skipif(not OpenEyeToolkitWrapper.toolkit_is_available(), reason='Test requires OE toolkit')
```

Comment:
Usually best to to generate this mark like so:

```python
using_oetk = pytest.mark.skipif( not(OpenEyeToolkitWrapper.toolkit_is_available()), reason='Test requires OE toolkit')

@using_oetk
def mytest():
    ...
```

Reply:
Nice. Will do, thanks!